packages/rules-unit-testing/src/api/index.ts

```diff
@@ -197,11 +197,7 @@ function createUnsecuredJwt(token: TokenOptions, projectId?: string): string {
 
   // Unsecured JWTs use the empty string as a signature.
   const signature = '';
-  return [
-    base64.encodeString(JSON.stringify(header), /*webSafe=*/ false),
```

Comment:
This needs use base64url instead of base64. https://github.com/firebase/firebase-js-sdk/blob/5d31e2192d0ea68a768bc7826ad5aa830c2bc36c/packages/util/src/crypt.ts#L339 is the utility for that. I think we still need to trim padding ourselves though.